packages/story-editor/src/components/storyFontPicker/index.js

```diff
@@ -78,6 +79,10 @@ const StoryFontPicker = forwardRef(function StoryFontPicker({ ...rest }, ref) {
 
   const onChange = useCallback(
     async (newFont) => {
+      trackEvent('font_family_changed', {
+        name: newFont.name,
+      });
```

Comment:
And this! May be that we just add 'text' statically
```suggestion
      trackEvent('font_family_changed', {
        name: newFont.name,
        type: 'text'
      });
```